docs/standards/hcs-2.md

```diff
@@ -48,7 +66,8 @@ The registry should adopt a standardized format to ensure consistent access and
 
 | Operation     | Description                                                        | Usable in non-indexed topic          |
 |-----------|--------------------------------------------------------------------|------------------------|
-| `Register` | Adds new entries or versions to the registry.| ✅              |
+| `Register` | Adds new entries or versions to the registry.| ✅   
+| `Migrate`  | Moves messages to a new Topic ID. Previous messages are invalidated and new state is computed from the new Topic.  | ✅         |           |
```

Comment:
Previous messages are archived and new state is computed from the new Topic. The new topic id needs to either replicate all the data, or create a snapshot with a pointer to the old topic id.